mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_get_job_log_entries_function.sql

```diff
@@ -91,16 +90,6 @@ FROM (
                     FROM journal_records
                     WHERE journal_records.job_execution_id = ''%1$s'') AS rec_titles
             ON rec_titles.source_id = records_actions.source_id AND rec_titles.title IS NOT NULL
```

Comment:
Its great that this complicated WHERE cause refactored to use simple Having construction, now its easier to read, thank you

Reply:
Thank you for review)